src/Hocon/Builder/HoconArrayBuilder.cs

```diff
@@ -0,0 +1,46 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="HoconImmutableArrayBuilder.cs" company="Akka.NET Project">
+//      Copyright (C) 2013 - 2020 .NET Foundation <https://github.com/akkadotnet/hocon>
+// </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Hocon
+{
+    public sealed class HoconArrayBuilder : List<HoconElement>
```

Comment:
Should this be public or `internal`?

Reply:
public, there's no danger to expose a builder pattern, user can use it if they wanted to make their own Hocon from scratch, the `HoconElement` elements themselves are immutable.